src/feed-queue.js

```diff
@@ -1,5 +1,7 @@
+const config = require('./config');
```

Comment:
Put this after bull require and remove const config. Lines 1 and 2 will look like 
const Bull = require('bull');
require('./config');

Reply:
Removed `const config = ` and moved it to be below bull require.